package.json

```diff
@@ -25,7 +25,13 @@
     "start": "grunt watchAll",
     "test": "grunt test",
     "toc": "doctoc",
-    "docs": "jsdoc -r src/js -d docs/api -c .jsdoc.json"
+    "docs": "jsdoc -r src/js -d docs/api -c .jsdoc.json",
+    "predev": "babel src/js -d es5",
+    "dev": "run-p babel browserify",
+    "dev-test": "run-p dev test-watch",
+    "babel": "babel -w src/js -d es5",
```

Comment:
I'm going to pull these out of this PR when getting ready to merge and make a separate PR out of it.